internal/pkg/debug/graph/graph.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package graph
+
+import (
+	"golang.org/x/tools/go/ssa"
+)
+
+type NodeRelationship int
+
+const (
```

Comment:
Consider an enum instead.
https://www.ribice.ba/golang-enums/

Reply:
Isn't this the way to write an `enum` in Go? Do you want me to use strings?

Reply:
My bad, I did not notice that you created a type for this.